resources/views/post/edit.blade.php

```diff
@@ -0,0 +1,44 @@
+@extends('layouts.app')
+
+@section('content')
+
+<!-- Bootstrap Boilerplate... -->
+
+<div class="panel-body">
+    <!-- Display Validation Errors -->
+
+    <!-- New Task Form -->
+    <form action="{{ route('posts.update',$post->id) }}" method="POST" class="form-horizontal">
+        @csrf
+        @method('PUT')
+
+        <!-- Task Name -->
+        <div class="form-group">
+            <label for="task" class="col-sm-3 control-label">{{ trans('post') }}</label>
+            <div class="col-sm-6">
+                <input type="text" name="name" id="task-name" class="form-control" value="{{ $post->name }}">
+                @error('name')
```

Comment:
tương tự, bên dưới có lỗi này thì sửa lại hết đi nhé 
```suggestion
                @error ('name')
```